middleware/current-page-path.js

```diff
@@ -1,5 +1,14 @@
 export default function(context) {
   const { req, route } = context
-  const host = req ? req.headers.host : window.location.host
+  const host = _getHostFromHeaders(req.headers)
+    ? req.headers.host
+    : window.location.host
+
```

Comment:
Est-ce qu'ici il ne faudrait pas : 
```suggestion
  const host = _getHostFromHeaders(req.headers)
    ? _getHostFromHeaders(headers) 
    : window.location.host

```
Dans le cas actuel, si on a headers['x-forwarded-host'], ça va validé la ternaire, et on va avoir req.headers.host en fin de compte j'ai l'impression.